src/proxy.js

```diff
@@ -0,0 +1,278 @@
+import { checkParamOrThrow } from 'apify-client/build/utils';
+import { ENV_VARS, LOCAL_ENV_VARS } from 'apify-shared/consts';
+import { APIFY_PROXY_VALUE_REGEX } from 'apify-shared/regexs';
+import { COUNTRY_CODE_REGEX } from './constants';
+import { apifyClient } from './utils';
+import { requestAsBrowser } from './utils_request';
+
+// CONSTANTS
+const PROTOCOL = 'http';
+const APIFY_PROXY_STATUS_URL = 'http://proxy.apify.com/?format=json';
+
+/**
+ * @typedef ProxyConfigurationOptions
+ * @property {Array} [groups] - An array of proxy groups to be used
+ *   by the [Apify Proxy](https://docs.apify.com/proxy).
+ * @property {string} [country] - Two letter country code according to ISO 3166-1 alpha-2.
```

Comment:
country => countryCode, otherwise it's confusing

Reply:
Renamed `country` to `countryCode` to keep it clear.